templates/content/stats.html.twig

```diff
@@ -0,0 +1,69 @@
+{% extends 'layout.html.twig' %}
+
+{% block content %}
+
+    <div class="jumbotron">
+        <h1>Ho, ho, ho, Here is the statistics of Secret Santa</h1>
+    </div>
+
+    <div class="container-charts">
+        <h3 class="graph-title">Number of Secret Santa per month</h3>
+        <div class="chart chart-month">
+            {% if statisticsCounter.month %}
+                {% set values = statisticsCounter.month %}
+                {% for value, key in values %}
```

Comment:
the naming seeming really odd to me. the regular syntaxt of for is  `for key, value in values`. Why are you using the opposite ?

Reply:
BTW, `values` is a really bad name. It does not mean anything.

Reply:
BTW I don't think you need an additional variable

Reply:
My bad, I made a mistake.
I change the name of `key`, `value` and `values`